code/modules/reagents/reagents/vox.dm

```diff
@@ -0,0 +1,106 @@
+// Solvent - used in vox chemistry and stomachs.
+/datum/reagent/acid/voxenzyme
+	name = "alkahest"
+	id = "voxenzyme"
+	description = "A seething slurry of pseudo-living proteins, acids and enzymes, utterly alien in composition and capable of dissolving almost anything."
+	taste_description = "salt and burning"
+	reagent_state = LIQUID
+	color = COLOR_CYAN
+	power = 12
+	meltdose = 1
+	scannable = FALSE
+
+// Vox slurry - used to fuel machines, feed vox and gear, and as an ingredient in the biofoundry
+/datum/reagent/toxin/voxslurry
+	name = "protoslurry"
+	id = "voxslurry"
+	taste_description = "coppery slime"
+	description = "A complex slurry of lipids, proteins, metal particulate and long polymer chains. Ubiquitous to vox-inhabited stations and equipment."
+	strength = 6
+	scannable = FALSE
+
+/datum/reagent/toxin/voxslurry/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(alien != IS_VOX)
+		return ..()
+	M.adjust_nutrition(12 * removed)
+	M.heal_organ_damage(0.5 * removed, 0.5 * removed)
+	M.add_chemical_effect(CE_BLOODRESTORE, 4 * removed)
+
+/datum/reagent/toxin/voxslurry/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
+	affect_blood(M, alien, removed)
+
+// General-purpose vox medication
+/datum/reagent/toxin/voxmeds
+	name = "repair gel"
+	id = "voxmeds"
+	taste_description = "fizzing sweetness"
+	description = "A complex serum composed of vox-manufactured nanomachines, complex protein chains and microscopic bundles of metallic fibers."
+	strength = 8
+	scannable = FALSE
+
+/datum/reagent/toxin/voxmeds/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
+	affect_blood(M, alien, removed)
+
+/datum/reagent/toxin/voxmeds/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+
+	if(alien != IS_VOX)
```

Comment:
Dubious about the ordering using a negative condition where everything else asserts the truthful comparison of `alien`, but probably fine here.